pkg/syncer/spec/mutators/secrets.go

```diff
@@ -21,6 +21,7 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/kubernetes/pkg/apis/core"
```

Comment:
don't use the internal api packages of kube, but k8s.io/api. Everything in the syncer must work against k8s.io/api and other non k8s.io/kubernetes repos.

Reply:
done